Clinical-Genomics/cg
# Verified species

How to test:
0.0 Install clinical-api branch on stage 
0.1 install cgweb branch stage, both on clinical-db
0.2 upgrade database with cg/scripts/add-organism-table.sql script
0.3 Change some organisms in clinical-api to be verified
1. add some samples to an Microbial orderform with some verified strains and some not
2. upload OF in OP
3. submit
Expected outcome:
Verified organism should appear in LIMS on the submitted samples
